app/components/EventItem/styles.css

```diff
@@ -48,6 +55,12 @@
   max-height: 80px;
   object-fit: contain;
   background: white;
+  margin: var(--spacing-sm) var(--spacing-md) var(--spacing-sm) 0;
+  transition: var(--easing-fast);
+}
+
+.eventItem:hover .companyLogo img {
+  border-radius: var(--border-radius-sm);
 }
```

Comment:
This feels kind of off - why are we adding border-radius to the images as well

Reply:
I tried both with and without, but personally felt with felt more natural. No other reason than that